packages/create-river-build-app/README.md

```diff
@@ -24,3 +40,15 @@ If you want to create a new app in a different directory, you can specify the di
 ```bash
 yarn create river-build-app my-app
 ```
+
+You can specify a template using the `-t` or `--template` flag:
+
+```bash
+yarn create river-build-app my-app --template react-ts
+```
+
+Available templates:
+
+- react-ts (default)
+- react
+- playground
```

Comment:
Can we make playground the default?

Reply:
nice catch! it is the default, but it was wrong in the readme 🤦‍♂️